lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -110,7 +110,7 @@ private function build($data) {
       'id' => isset($northstar_user) ? $northstar_user->id : $data->field_northstar_id_value,
       'first_name' => dosomething_helpers_extract_field_data($drupal_user->field_first_name),
       'last_initial' => $last_initial,
-      'photo' => dosomething_helpers_extract_field_data($drupal_user->photo),
+      'photo' => isset($drupal_user->picture) ? $drupal_user->picture : NULL,
```

Comment:
@DFurnes @weerd I'm a little confused about how this would play out.
@weerd - you said "Kinda like how with Gladiator, we send the /signups request to Gladiator and it will convert all the NS IDs we pass to Phoenix IDs to send to Phoenix." - can you send me the code for this so I can see by example to wrap my head around?